Dynamic_Bar.lua

```diff
@@ -8,9 +8,7 @@ local Type = TMW.Classes.IconType:New("dynamicbar")
 Type.name = "Dynamic Bar"
 Type.desc = "A Bar that can be controlled by LUA scripts"
 Type.menuIcon = "Interface/Icons/inv_box_04"
-Type.unitType = "unitid"
 Type.hasNoGCD = true
-Type.canControlGroup = true
```

Comment:
This type isn't coded to be a group controller, and I can't imagine how that would ever fit in to something like this. There's other code below that I removed with this in mind.